packages/web-components/src/components/rux-input/rux-input.tsx

```diff
@@ -230,7 +259,15 @@ export class RuxInput implements FormFieldInterface {
                     <input
                         name={name}
                         disabled={disabled}
-                        type={type}
+                        //if type isn't password, go with type. If type is password and
+                        //the password should be visibile, then go with text. Otherwise, type.
+                        type={
```

Comment:
I'm thinking about making this a function that returns type instead of this nested ternary since it looks messy. What do you think?